.github/workflows/runE2ETest.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         nodeVersion:
           - 18.18.2
         vscodeVersion:
-          - stable
+          - 1.85.2
```

Comment:
Can we leave a comment here as to why we are pinning this version and also a link to the wdio bug so we can keep track of it?

Reply:
Bug that @peternhale filed in the WDIO repo: https://github.com/webdriverio/webdriverio/issues/12167

We are pinning this version because VSCode 1.85.2 uses Chromedriver 114, which is the last Chromedriver version that is supported by the old URL format, which WDIO calls to get the endpoint.

The support person will have to monitor the [PR in WDIO](https://github.com/webdriverio-community/wdio-vscode-service/pull/94) to see when WDIO fixes the bug.  At that point, we can upgrade our VSCode version for E2E tests back to `latest`.

Reply:
I mean can we leave a comment in code for future reference?

Reply:
https://github.com/forcedotcom/salesforcedx-vscode/pull/5399